technitium-dns/docker-compose.yml

```diff
@@ -0,0 +1,46 @@
+version: "3.7"
+
+services:
+  app_proxy:
+    environment:
+      APP_HOST: technitium-dns_dns-server_1
+      APP_PORT: 5380
+  dns-server:
+    image: technitium/dns-server:12.1@sha256:f5de7eea6dd85f85656a1ea7584c6af0619bb8b1fa83e5c7992fcf38b76c7ccf
+
+    ports:
+      # (proxied) - "5380:5380/tcp" #DNS web console (HTTP)
+      # - "53443:53443/tcp" #DNS web console (HTTPS)
+      - "53:53/udp" #DNS service
+      - "53:53/tcp" #DNS service
+      # - "853:853/udp" #DNS-over-QUIC service
+      # - "853:853/tcp" #DNS-over-TLS service
+      # - "443:443/udp" #DNS-over-HTTPS service (HTTP/3)
+      # - "443:443/tcp" #DNS-over-HTTPS service (HTTP/1.1, HTTP/2)
+      # - "80:80/tcp" #DNS-over-HTTP service (use with reverse proxy or certbot certificate renewal)
+      # - "8053:8053/tcp" #DNS-over-HTTP service (use with reverse proxy)
+      # - "67:67/udp" #DHCP service      
```

Comment:
What do you think about adding other ports? Obviously we don't have inbuilt https on umbrelOS quite yet, but users may want to use technitium dns as a DHCP server for example?

Actually, thinking about it I wonder if we need to run technitium dns in host network mode for now to get the full capabilities of the app (like we do with pi-hole in order to get the DHCP server functionality working).

What do you think?

Reply:
OK, since it's done with network_mode: "host" in pi-hole, I'll replicate that here

Reply:
Sounds good, I think that is the right call in order to make Technitium DNS useful.

FYI, I have made the following additional changes so that it can work with host networking:
- removed the app proxy: https://github.com/getumbrel/umbrel-apps/pull/1046/commits/ddf54df6ff67efdd81948cfcfce3971ce813f14a
- removed the sysctls port range: https://github.com/getumbrel/umbrel-apps/pull/1046/commits/d69edd429b2285250aed1f035fc1e00b89bcea5b